.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
         ".eslintcache": true,
         "yarn-error.log": true,
         "app.log": true
-    }
+    },
+    "eslint.enable": true
 }
```

Comment:
:art: if you're using VSCode I'd recommend adding [this extension](https://marketplace.visualstudio.com/items?itemName=samverschueren.final-newline) to catch these missing final newlines.